File: ITManagementClient/Models/ResponseModels/ProjectWorkers/GetListOfAvailableDevelopersResponseModel.cs
using System.Collections.Generic;
using ITManagementClient.Models.ResponseModels.Workers;

namespace ITManagementClient.Models.ResponseModels.ProjectWorkers
{
    public class GetListOfAvailableDevelopersResponseModel
    {
        public IEnumerable<WorkerGridResponseModel> AvailableDevelopers { get; set; }
    }
}

File: ITManagementServer/src/main/java/models/responseModels/projectWorkers/GetListOfAvailableDevelopersResponseModel.java
package models.responseModels.projectWorkers;

import models.responseModels.workers.WorkerGridResponseModel;

import java.util.ArrayList;
import java.util.List;

public class GetListOfAvailableDevelopersResponseModel {
    private List<WorkerGridResponseModel> AvailableDevelopers;

    public GetListOfAvailableDevelopersResponseModel() {
        AvailableDevelopers = new ArrayList<>();
    }

    public List<WorkerGridResponseModel> getAvailableDevelopers() {
        return AvailableDevelopers;
    }

    public void setAvailableDevelopers(List<WorkerGridResponseModel> availableDevelopers) {
        AvailableDevelopers = availableDevelopers;
    }
}